Server/Plugins/APIDump/APIDesc.lua

```diff
@@ -11653,6 +11653,10 @@ a_Player:OpenWindow(Window);
 				{
 					Notes = "Saves all the chunks in all the worlds. Note that the saving is queued on each world's tick thread and this functions returns before the chunks are actually saved.",
 				},
+				SaveAllChunksNow =
```

Comment:
I would hold off exporting this, it could introduce deadlocks if plugins try to use it in a world thread